inscriptions/ISic000543.xml

```diff
@@ -202,8 +220,10 @@
                 <app loc="line 5"><note>Auria 1697: [..]IGR[..]NVS[..]; Gualtherus 1624: ΛIGRANIVS</note></app>
```

Comment:
which leaves us with an interesting question, as to whether we think it's Nigranius or Nigrianus; the latter is however better attested from a quick look in Clauss Slaby.